CHANGELOG.md

```diff
@@ -5,7 +5,6 @@
 ### Grafana Mimir
 
 * [CHANGE] The following deprecated configurations have been removed: #6673
-  * `-querier.query-ingesters-within`
```

Comment:
Ah, I got confused about what was removed in https://github.com/grafana/mimir/pull/6673/commits/1d6581c6a07d8f851990bacbe0cfdd7186a89de9; it was the `query_ingesters_within` YAML field, not this flag, which already got moved in https://github.com/grafana/mimir/pull/4287.

Should we add to the change log about the removed YAML field instead?

Reply:
yes. i think so. I'll merge this PR